pkg/kev/kev.go

```diff
@@ -29,10 +29,14 @@ import (
 	"github.com/pkg/errors"
 )
 
-const (
+var (
 	// ManifestName main application manifest
 	ManifestName = "kev.yaml"
```

Comment:
How about if we change this to `ManifestFilename`?

Then `symbol` in `MintEnvironments` fun can simply be called `manifestName`?